### PR TITLE
Add SMI extra packages

### PR DESCRIPTION
**Pull request by mrakitin**
Added SMI extra packages (`peakutils` at the moment).
@licode

### CODE REVIEW

**Comment by jrmlhermitte** [comment]
lgtm

**Comment by mrakitin** [comment]
Thanks for merging

File: recipes-tag/12-id-smi-analysis/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+{% set year = "2018" %}
+{% set cycle = "2" %}
+{% set version ="0" %}
+
+package:
+  name: 12-id-smi-analysis
+  version: {{ year }}-{{ cycle }}.{{ version }}
```
Comment (licode) [inline recipes-tag/12-id-smi-analysis/meta.yaml]: conda doesn't allow "-" in version number. Can you correct this, we use C for other pkgs. https://github.com/NSLS-II/lightsource2-recipes/blob/master/recipes-tag/07-bm-qas-collection/meta.yaml 



Error: bad character '-' in package/version: 2018-2.0
 raised with check_output comand ['conda', 'build', '/repo/recipes-tag/12-id-smi-analysis', '--output', '--python', '3.5', '--numpy', '1.11']
['conda', 'build', '/repo/recipes-tag/12-id-smi-collection', '--output', '--python', '3.5', '--numpy', '1.11']
Error: bad character '-' in package/version: 2018-2.0

File: recipes-tag/12-id-smi-collection/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+{% set year = "2018" %}
+{% set cycle = "2" %}
+{% set version ="0" %}
+
+package:
+  name: 12-id-smi-collection
+  version: {{ year }}-{{ cycle }}.{{ version }}
```
Comment (licode) [inline recipes-tag/12-id-smi-collection/meta.yaml]: same here.

Comment (mrakitin) [reply recipes-tag/12-id-smi-collection/meta.yaml]: Thanks @licode for the hint! Do we want another PR taking care of it? Or reverting and fixing may be better?

Comment (licode) [reply recipes-tag/12-id-smi-collection/meta.yaml]: I will revert it.

**Pull request merged**